### PR TITLE
docs: add tls configurations

### LINKED ISSUE #815: Update docs for feat: add configuration for tls watch option
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/3395

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

*Describe the change in this PR*

TLS configuration

Fixes #815 

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by nicecui — APPROVED** [review]

File: versioned_docs/version-0.9/user-guide/deployments/configuration.md [hunk 1]
```diff
@@ -179,6 +179,17 @@ The following table describes the options in detail:
...
+For MySQL and Postgres interface, TLS can be configured to enable transport
+layer security.
+
+| Option                        | Key         | Type    | Description                                                   |
+|-------------------------------|-------------|---------|---------------------------------------------------------------|
+| `mysql.tls` or `postgres.tls` |             |         | TLS configuration for MySQL and Postgres                      |
+|                               | `mode`      | String  | TLS mode, options are `disable`, `prefer` and `require`       |
+|                               | `cert_path` | String  | File path for TLS certificate                                 |
+|                               | `key_path`  | String  | File path for TLS private key                                 |
+|                               | `watch`     | boolean | Watch file system changes and reload certificate and key file |
```
Comment (fengjiachun) [inline versioned_docs/version-0.9/user-guide/deployments/configuration.md:191]:
```suggestion
|                               | `watch`     | Boolean | Watch file system changes and reload certificate and key file |
```

File: i18n/zh/docusaurus-plugin-content-docs/version-0.9/user-guide/deployments/configuration.md [hunk 2]
```diff
@@ -169,6 +169,16 @@ enable = true
...
 
+对 MySQL 和 Postgres 接口，我们支持 TLS 配置
+
+| Option                        | Key         | Type    | Description                                      |
+|-------------------------------|-------------|---------|--------------------------------------------------|
+| `mysql.tls` 或 `postgres.tls` |             |         | MySQL 或 Postgres 的 TLS 配置                    |
+|                               | `mode`      | String  | TLS 模式，支持 `disable`, `prefer` and `require` |
+|                               | `cert_path` | String  | TLS 证书文件路径                                 |
+|                               | `key_path`  | String  | TLS 私钥文件路径                                 |
+|                               | `watch`     | boolean | 监控文件变化，自动重新加载证书或私钥             |
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/version-0.9/user-guide/deployments/configuration.md:180]:
```suggestion
|                               | `watch`     | Boolean | 监控文件变化，自动重新加载证书或私钥             |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments/configuration.md [hunk 3]
```diff
@@ -169,6 +169,17 @@ enable = true
...
 
+对 MySQL 和 Postgres 接口，我们支持 TLS 配置
+
+| Option                        | Key         | Type    | Description                                      |
+|-------------------------------|-------------|---------|--------------------------------------------------|
+| `mysql.tls` 或 `postgres.tls` |             |         | MySQL 或 Postgres 的 TLS 配置                    |
+|                               | `mode`      | String  | TLS 模式，支持 `disable`, `prefer` and `require` |
+|                               | `cert_path` | String  | TLS 证书文件路径                                 |
+|                               | `key_path`  | String  | TLS 私钥文件路径                                 |
+|                               | `watch`     | boolean | 监控文件变化，自动重新加载证书或私钥             |
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments/configuration.md:180]:
```suggestion
|                               | `watch`     | Boolean | 监控文件变化，自动重新加载证书或私钥             |
```

File: docs/user-guide/deployments/configuration.md [hunk 4]
```diff
@@ -179,6 +181,17 @@ The following table describes the options in detail:
...
+For MySQL and Postgres interface, TLS can be configured to enable transport
+layer security.
+
+| Option                        | Key         | Type    | Description                                                   |
+|-------------------------------|-------------|---------|---------------------------------------------------------------|
+| `mysql.tls` or `postgres.tls` |             |         | TLS configuration for MySQL and Postgres                      |
+|                               | `mode`      | String  | TLS mode, options are `disable`, `prefer` and `require`       |
+|                               | `cert_path` | String  | File path for TLS certificate                                 |
+|                               | `key_path`  | String  | File path for TLS private key                                 |
+|                               | `watch`     | boolean | Watch file system changes and reload certificate and key file |
```
Comment (fengjiachun) [inline docs/user-guide/deployments/configuration.md:193]:
```suggestion
|                               | `watch`     | Boolean | Watch file system changes and reload certificate and key file |
```

**Review by fengjiachun — APPROVED** [review]
Rest LGTM

**Pull request merged**